src/test/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpointIT.java

```diff
@@ -153,4 +154,70 @@ private void createIACStub() throws IOException {
                     .withHeader("Content-Type", "application/json")
                     .withBody("[\"grtt7x2nhygg\"]")));
   }
+
+  /**
+   * Sends a sample unit in a message so that casesvc creates a case, then waits for a message on
+   * the case lifecycle queue which confirms case creation
+   *
+   * @return a new CaseNotification
+   */
+  private CaseNotification sendCaseWaitForNotification(
```

Comment:
This method is sort of long, looks like it's doing two things, sending a sample unit and waiting for a case notification, maybe split these into two methods.